### ISSUE TITLE
Asset selector should not prompt to sign-in if already signed in

### ISSUE THREAD

**Issue by sdmcraft** [hlxsites/aem-assets-boilerplate#27]
Currently if we refresh the authoring tab in browser and reopen the Asset Selector, it prompts to sign in again. Although the user gets directly logged in on clicking "Sign In" button instead of going through the SUSI flow. But this can be improved by not asking the user to sign in.

**Comment by mfrisbey** [comment]
I played around with this, and found a solution that seems to work. In summary, the auth service's [onAccessTokenReceived](https://github.com/mfrisbey/franklin-tutorial/blob/08782c367e3eec7615c847d572ae757be93c6cd5/blocks/aem-asset-selector/aem-asset-selector-util.js#L37) method seems like a good way to know whether the user is logged in already.

So, when the page loads, I added a delay before showing the user the "Sign In" button. If we receive the `onAccessTokenReceived` event before that delay is complete, we skip showing the sign in button and just display the asset selector.
